### PR TITLE
[DW-0000] POC new folder strucure

### PR DESCRIPTION
**Pull request by giacomolm**

### CODE REVIEW

File: site/src/main/java/uk/nhs/digital/ps/linkprocessors/DefaultDocumentLinkProcessor.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package uk.nhs.digital.ps.linkprocessors;
```
Comment (jacsamell) [inline site/src/main/java/uk/nhs/digital/ps/linkprocessors/DefaultDocumentLinkProcessor.java]: Move out of publication system (ps) package

File: site/src/main/java/uk/nhs/digital/ps/sitemaphandlers/DefaultDocumentSitemapHandler.java [hunk 2]
```diff
@@ -0,0 +1,112 @@
+package uk.nhs.digital.ps.sitemaphandlers;
```
Comment (jacsamell) [inline site/src/main/java/uk/nhs/digital/ps/sitemaphandlers/DefaultDocumentSitemapHandler.java]: Move out of publication system (ps) package

File: site/src/main/java/uk/nhs/digital/ps/linkprocessors/DefaultDocumentLinkProcessor.java (hunk 1 above)
Comment (giacomolm) [reply site/src/main/java/uk/nhs/digital/ps/linkprocessors/DefaultDocumentLinkProcessor.java]: Thanks for the comment. I will move it to the common package

File: site/src/main/java/uk/nhs/digital/ps/sitemaphandlers/DefaultDocumentSitemapHandler.java (hunk 2 above)
Comment (giacomolm) [reply site/src/main/java/uk/nhs/digital/ps/sitemaphandlers/DefaultDocumentSitemapHandler.java]: Thanks for the comment. I will move it to the common package

**Comment by giacomolm** [comment]
Changes completed!

**Pull request merged**